kolibri/plugins/epub_viewer/assets/src/views/EpubRendererIndex.vue

```diff
@@ -318,8 +318,8 @@
           h5: { ...colorStyle },
           'p:first-of-type::first-letter': { ...colorStyle },
           // help media not overflow their columns
-          video: { 'max-width': '100%' },
-          img: { 'max-width': '100%' },
+          video: { 'max-width': '100vw' },
+          img: { 'max-width': '100vw' },
```

Comment:
I don't believe this to work in all cases. Some epubs may have 2 page, book-like, layouts where pages show side-by-side. Having this go from `100%`, which would be 100% of the containing element's width, to `100vw` would change it to be the browser's full viewport/frame width. Therefore, I think this should remain as is.